### PR TITLE
Add Docker forwarder information

### PR DESCRIPTION
**Pull request by perk**

### CODE REVIEW

File: docker.adoc [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+The Docker containers send logs using docker logs driver into fluentd forwarder which takes care of the logs distribution.
+
+Please consult the <<fluentd-container.adoc#, Docker `fluentd` forwarder>> on how to set up the forwarder.
+
+
+=== How to push Docker containers logs into the Logsense forwarder
+
+To push the container's logs into the forwarder the Docker logging driver needs to be configured. We can do it by providing the `--log-driver` and `--log-opt` flags.
+
+----
```
Comment (pmaciolek) [inline docker.adoc:21]: `[source,bash]` ?

File: fluentd-container.adoc [hunk 2]
```diff
@@ -0,0 +1,32 @@
+= Docker `fluentd` forwarder image
+
+The following https://hub.docker.com/r/collectivesense/logsense-forwarder[Logsense forwarder image] can be used to set up your `fluentd` forwarder as Docker container.
+
+The source code can be found in our https://github.com/collectivesense/logsense-fluentd-forwarder[GitHub repository]
+
+It can be run as follow:
+
+----
```
Comment (pmaciolek) [inline fluentd-container.adoc:12]: `[source,bash]` ?

File: fluentd-container.adoc [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  --env LOGSENSE_TOKEN="your logsense token" \
+  -p 127.0.0.1:24224:24224 \
+  collectivesense/logsense-forwarder
+----
+
+Please remember to replace the `"your logsense token"` string with your token.
+
+It can also be run using Docker Compose with the following yaml file:
+
+----
```
Comment (pmaciolek) [inline fluentd-container.adoc:24]: `[source,yaml]` ?

**Review by pmaciolek — APPROVED** [review]
Some minor comments

I am not sure about docs not linked from the main site

File: fluentd-container.adoc (hunk 2 above)
Comment (perk) [reply fluentd-container.adoc:12]: :+1:

**Pull request merged**